### PR TITLE
5th Patrons

### PR DESCRIPTION
**Pull request by zumbak04**
## Developer changelog:
- Added patron characters for:
  - Spartan, ingame character is Edward Janeway, count.
  - Daruthyr, ingame character is Gyliad MacColach, count.
  - BadManCZ, ingame character is Sedrieth Forestcaller, count.
  - Adam M, ingame character is Glarr'glarr Glarr, count.
  - Wiggles The Sea Turtle, ingame character is Willis Winter, count.

### CODE REVIEW

**Comment by Gartarr** [comment]
Edward Janeway (310400) misses Ravager trait, Gyliad (310410) has one eye too much (supposed to have one_eyed trait in files, but doesn't have it in-game, same situation as Edward), also, he is a twin, but has no siblings, whatever i guess?
Also, Sedrieth (310420) and Glarr'glarr (310430) miss
disallow_random_traits = yes
so, random traits added on top of these specified, don't know if this is intended.

File: history/characters/310000_patron_characters.txt [hunk 1]
```diff
@@ -835,4 +835,104 @@
...
+	effect={add_patron_trait_character_effect=yes}
+	add_trait=legit_bastard add_trait=twin add_trait=lefthanded
+	add_trait=fair
+	disallow_random_traits = yes
+	560.1.27={ birth=yes add_trait=creature_human }
+	580.1.1={
+		add_trait=brilliant_strategist
+		add_trait=class_warrior_6
+		add_trait=mystic
+		add_trait=scarred_mid add_trait=one_eye add_trait=lustful add_trait=wroth add_trait=brave add_trait=paranoid add_trait=stubborn add_trait=zealous add_trait=unyielding_leader add_trait=rough_terrain_leader
```
Comment (Gartarr) [inline history/characters/310000_patron_characters.txt]: one_eyed, not one_eye :)

Comment (Gartarr) [inline history/characters/310000_patron_characters.txt]: one_eyed, not one_eye :)

File: history/characters/310000_patron_characters.txt [hunk 2]
```diff
@@ -835,4 +835,104 @@
...
+	martial=5 diplomacy=5 stewardship=5 intrigue=5 learning=5
+	effect={add_patron_trait_character_effect=yes}
+	disallow_random_traits = yes
+	559.12.15={ birth=yes add_trait=creature_human }
+	580.1.1={
+		add_trait=intricate_webweaver
+		add_trait=class_rogue_6
+		add_trait=shrewd
+		add_trait=strategist
+		add_trait=falconer add_trait=ambitious add_trait=proud add_trait=stubborn add_trait=greedy add_trait=envious add_trait=cynical add_trait=patient add_trait=ravager
```
Comment (Gartarr) [inline history/characters/310000_patron_characters.txt]: This seems correct, but for some reason, Ravager isn't there. I would check requirement for trait:
potential = {
		OR = {
			# Warcraft
			religion_group = vrykul_religion_group
			culture = pirate_culture
			# religion = norse_pagan
			# religion = norse_pagan_reformed
			
			any_owned_bloodline = {
				has_bloodline_flag = bloodline_viking
			}
			AND = {
				has_religion_feature = religion_adventuring
				has_religion_feature = religion_seafaring
			}
		}
	}
If I change his religion in this file to for. ex. Odyn, which belongs to vrykul_religion_group, it works, but only as main religion, secret doesn't affect it.

File: history/characters/310000_patron_characters.txt (hunk 1 above)
Comment (zumbak04) [reply history/characters/310000_patron_characters.txt]: Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/f37100e301f5cb38789cb7f76506f9d0e4a4e718.

**Comment by zumbak04** [comment]
Hello! Thanks for review!

> he is a twin, but has no siblings, whatever i guess?
 
The patron allowed the removal, so removed the trait.

> Also, Sedrieth (310420) and Glarr'glarr (310430) miss
> disallow_random_traits = yes
> so, random traits added on top of these specified, don't know if this is intended.

It's intended.

**Comment by MemeWizard** [comment]
> he is a twin, but has no siblings, whatever i guess?

Btw, just like Thomas the Apostle in vanilla

File: history/characters/310000_patron_characters.txt (hunk 2 above)
Comment (zumbak04) [reply history/characters/310000_patron_characters.txt]: Nah, it was a leftover after the testing... unless we want Gilnean to be sea raiders.
Cleared it in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/7a27f80a0148262ccbbd6942118b3d6fb3799405.

**Pull request merged**